packages/kyt-starter-universal/.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-  "presets": ["kyt-react"]
+  "presets": ["kyt-react"],
+  "plugins": ["syntax-dynamic-import"]
```

Comment:
Do you think we should just add this plugin to our babel-preset-kyt-core? This seems like something kyt should support/recommend out of the box.